practice/validator/site.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+// 1. Input
+/**
+ * Gathers what has been typed in the 'name' field for each event.
+ */
+function getUserName() {
+  return $('#name').val();
+}
+/**
+ * Gathers what has been typed in the 'DOB' field for each event.
+ */
+function getUserDOB() {
+  return $('#dob').val();
+}
+/**
+ * Gathers what has been typed in the 'Phone' field for each event.
+ */
+function getUserPhone() {
+  return $('#phone').val();
+}
+
+// 2. Transform Functions
+// No transform.
+
+// 3. Create
+var NAME_MATCH = /\w+\s\w+/;
+var DOB_MATCH = /\d\d\d\d-\d\d-\d\d/;
```

Comment:
I believe \w will match a-z, A-Z and 0-9.  An alternative is [a-zA-Z] or [a-z] with an 'i' flag, making it case-insensitive.